Add cobalt

ben-pearce opened the pull request:
Adds [cobalt](https://github.com/wukko/cobalt) media downloader tool

walkxcode reviewed (changes requested):
Please add a light version and SVG if possible per [guidelines](https://github.com/walkxcode/dashboard-icons/blob/main/CONTRIBUTING.md).

ben-pearce commented:
Added a light version which is a simple color invert of the original. The author of cobalt does not provide an official light version.

Added SVGs as requested and appropriately named.

walkxcode commented:
> [!WARNING]
> The repository has been migrated from `walkxcode` to `homarr-labs`, because I do not have the capacity/time to keep maintaining it. The Homarr team will take over all management and maintenance, whilst keeping all functionality for you, the user, the same. The project will always be usable outside of Homarr itself and breaking changes will not be made. ^Bjorn
>
> Due to a force merge into the main branch, all PRs have unfortunately been affected. We apologize for the inconvenience. Please recreate a new PR from the current `main` branch.
>
> The license and guidelines have changed, so please review them. If you want to help with maintenance, reach out to [homarr-labs@proton.me](mailto:homarr-labs@proton.me).
